charts/src/main/java/hu/ma/charts/line/LineChart.kt

```diff
@@ -67,7 +67,8 @@ fun LineChart(
     ) -> Unit
   )? = { position, entries ->
     LinesChartLegend(position = position, entries = entries)
-  }
+  },
+  updateOnDrag: Boolean = false,
```

Comment:
Could the naming of the variable be more expressive? Suggestion: callDrilldownCallbackOnDrag